sw/tools/generators/gen_flight_plan.ml

```diff
@@ -940,6 +940,7 @@ let () =
       check_altitude (float_of_string alt) xml;
       check_altitude_srtm (float_of_string alt) xml !fp_wgs84;
 
+      Xml2h.define "NOMINAL_ALT" (sprintf "%.0f /* nominal altitude of the flight plan */" (float_of_string alt));
```

Comment:
I would prefer `NAV_ALT` or `NAV_DEFAULT_ALT` for consistency with other defines